src/main/java/lotto/domain/LottoResults.java

```diff
@@ -0,0 +1,28 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class LottoResults {
+    private static final int PERCENT = 100;
+
+    private final Map<MatchResult, Integer> lottoResults;
+
+    public LottoResults(Map<MatchResult, Integer> matchResults) {
```

Comment:
```suggestion
    public LottoResults(WinningNumbers winningNumbers, BonusNumber bonusNumber) {
```
이렇게 바꿔보면 어떨까요?
컬렉션을 외부에서 받아오지 말고 LottoResults가 일급 콜렉션이니 외부에서 받아온 값들로 로또 결과를 만들 수 있지 않을까요?